apps/examples/board_specific_demo/ameba_mipi_demo/Kconfig

```diff
@@ -0,0 +1,14 @@
+#
+# For a description of the syntax of this configuration file,
+# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
+#
+
+config EXAMPLES_MIPI
+	bool "MIPI example"
+	default n
+	---help---
+		Enable MIPI Example
```

Comment:
Let me suggest below
```
config EXAMPLES_AMEBA_MIPI
       bool "MIPI example"
       default n
       select AMEBASMART_MIPI
....
```

1. This is not a general mipi example so that I suggest adding `AMEBA` at the config name.
2. To work this example, `CONFIG_AMEBASMART_MIPI` should be turned on. So I suggest adding `select` for dependency.